zproject/prod_settings_template.py

```diff
@@ -821,6 +821,14 @@
 ## from gravatar.com.
 # ENABLE_GRAVATAR = True
 
+## Overrides the above setting for individual realms
+## due to performance reasons, the realm_id has to be used
+## you can show a list of realms using this command: ./manage.py list_realms
+# GRAVATAR_REALM_OVERRIDE = {
+#   1 : True, # enable for first realm
+#   2 : False, # disable for other realm
+# }
```

Comment:
I don't think this is generally relevant enough to document here; we can comment in `default_settings.py`.